src/api/app/controllers/webui/package_controller.rb

```diff
@@ -103,6 +103,7 @@ def dependency
       redirect_back(fallback_location: project_show_path(project: params[:dproject]))
       return
     end
+
```

Comment:
I guess this commit's message is totally unrelated to what changes you added...?? :thinking: